tests/next/include/helpers.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Create two network namespaces, with a veth pair. Configuration is the
+# following:
+# ns0 - veth01 - 10.0.42.1/24 - 1111::1/64
+# ns1 - veth10 - 10.0.42.2/24 - 1111::2/64
+two_ns_simple() {
+	add_ns ns0 ns1
+	add_veth_pair veth01 ns0 veth10 ns1
+	ip -net ns0 address add 10.0.42.1/24 dev veth01
+	ip -net ns1 address add 10.0.42.2/24 dev veth10
+	ip -net ns0 address add 1111::1/64 dev veth01
+	ip -net ns1 address add 1111::2/64 dev veth10
+}
+
+# On top of two_ns_simple adds one VLAN interface per network namespace, as
+# follow:
+# ns0 - veth01.123 - 10.0.43.1/24 - 2222::1/64
+# ns1 - veth10.123 - 10.0.43.2/24 - 2222::2/64 - VLAN h/w offload off
```

Comment:
Why the asymmetric VLAN h/w offload switching? We don't have any existing test that requires this. While I do see it as a good scenario for a test, I don't see this setup as being so common to justify it being in "helpers".